vendor/bat-native-ledger/src/bat/ledger/internal/state/state.cc

```diff
@@ -53,6 +54,43 @@ std::vector<double> StringToVectorDouble(const std::string& items_string) {
   return items;
 }
 
+std::string PayoutStatusToString(
+    const base::flat_map<std::string, std::string>& payout_status) {
+  base::Value::Dict dict;
+  for (const auto& status : payout_status) {
+    dict.Set(status.first, status.second);
+  }
+
+  std::string payout_status_string;
+  base::JSONWriter::Write(dict, &payout_status_string);
+
+  return payout_status_string;
+}
+
+base::flat_map<std::string, std::string> StringToPayoutStatus(
+    const std::string& payout_status_string) {
+  absl::optional<base::Value> dict =
+      base::JSONReader::Read(payout_status_string);
+  if (!dict || !dict->is_dict()) {
```

Comment:
`Value::is_dict()` is checked internally by `Value::GetAsDictionary()`.
So this can be:
```
base::DictionaryValue* dictionary_value = nullptr;
auto dict = base::JSONReader::Read(payout_status_string);
if (!dict || !dict->GetAsDictionary(&dictionary_value)) {
  return {};
}
DCHECK(dictionary_value);
```

Reply:
Nice, fixed.